### ISSUE TITLE
Xdebug

### ISSUE THREAD

**Issue by villers** [woprrr/symfony-4-skeleton-docker#2]
Hello,

Can you explain how to configure phpstorm with xdebug?

Regards,
VILLERS Mickaël

**Comment by villers** [comment]
Okey i find it.

docker-compose.yml
```
php:
    environment:
        PHP_IDE_CONFIG: "serverName=ServerName"
```

I Install php docker extension in phpstorm and configure php interpreter for use the container php.

And  i configure xdebug server with Host: ServerName, Host: php, Port: 80 and mapping with /Projects/symfony-4-skeleton-docker/symfony | /app/symfony

**Comment by woprrr** [comment]
@villers Yes you right I haven't lot of time ATM to add support and explanations to use Xdebug. I have added the PHP_IDE_CONFIG env key and write (faster) a beginning of documentation to show how to add Interpretter / Server / configure Remote Xdebug server.

https://github.com/woprrr/symfony-4-skeleton-docker/blob/master/doc/phpstorm-macosx.md

https://github.com/woprrr/symfony-4-skeleton-docker#configure-xdebug

Do not hesitate to propose a re-reading of the documentation I did it very quickly after your request.
